_posts/2017-11-03-suds.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: SUDS
+date: 2017-11-2
+categories:
+  - projects
+description: Shower Use Detection System
+image: https://csh.rit.edu/~bigdata/SUDS.jpg
+image-sm: https://csh.rit.edu/~bigdata/SUDS.jpg
+author: Seth Gower
+author-image: https://avatars3.githubusercontent.com/u/19225089
+author-bio: First year Computer Engineering student
+author-email: seth@sethgower.com
+author-social:
+  github: https://github.com/SethGower
+  linkedin: https://www.linkedin.com/in/sethgower/
+---
+[SUDS](https://github.com/sethgower/suds), or the Shower Use Detection System, shows the status of the showers in our dorm hall, which is located on the third floor of Nathaniel Rochester Hall. When someone is in the shower and the door is locked, an LED on a panel in the hall next to the bathroom illuminates. In addition to the LEDs outside of the bathroom, I will be creating a web app to track live status updates remotely, and potentially collect statistics on when the showers/stalls are most used.
+
+SUDS is the first big hardware project that I have ever actually done, and the first time I have soldered my own board (outside of soldering headers onto MCUs and chips). The web app that I am going to build will be my first time using a MySQL database and creating a server-side backend that handles all of the input from the bathrooms.
+
+SUDS is a project that was first created by CSHers a few years ago. It was originally a simple circuit driven by AA batteries, but my version runs on a Raspberry Pi. 
+
+SUDS is driven by a Raspberry Pi in the bathroom ceiling, which outputs 5V; for future versions I will be making it . The current then goes to a limit switch on the door lock which closes the circuit and illuminates the LED for that shower. The wiring diagrams can be found on the [GitHub page](https://github.com/sethgower/suds). On the proto-board, on the same row as the anode of the LED, there is a pulldown resistor that leads to the GPIO pin corresponding to that shower.
```

Comment:
>for future versions I will be making it . 

I assume you left this part out, lol.

Reply:
Other than this, lgtm

Reply:
done

Reply:
where? I see the merge commit, but still see the error. I also suggest squashing all your commits before we merge this.

Reply:
check again I was trying to update my dev branch.